src/accessControl/acp/index.js

```diff
@@ -221,14 +262,56 @@ export default class AcpAccessControlStrategy {
 
   #policyUrl;
 
+  #policiesContainer;
+
   #fetch;
 
   constructor(originalWithAcr, policiesContainer, fetch) {
     this.#originalWithAcr = originalWithAcr;
+    this.#policiesContainer = policiesContainer;
     this.#policyUrl = getPolicyUrl(originalWithAcr, policiesContainer);
     this.#fetch = fetch;
   }
 
+  async getPermissionsForNamedPolicies(policyName) {
```

Comment:
I'm very unsure of the use of classes and AccessControl. They can be very hard to read, and I'm thinking we might want to introduce a model module for this logic instead. But let's discuss this with @ajacksified before doing any changes.